src/Events/StartMeasuring.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace AG\ElasticApmLaravel\Events;
+
+class StartMeasuring
+{
+    /** @var string */
+    public $name;
+
+    /** @var string */
+    public $type;
+
+    /** @var string|null */
+    public $action;
+
+    /** @var string|null */
+    public $label;
+
+    /** @var float|null */
+    public $startTime;
```

Comment:
Can you define variables as `snake_case`? I know we have a bit of a mess now with code standards, but I'm working on it https://github.com/arkaitzgarro/elastic-apm-laravel/pull/15.

Reply:
Well, Symfony coding standards expect variables in `camelCase` and configuration in `snake_case`. It is basically the opposite of what we are doing 😅